### ISSUE TITLE
Loadout Page Is Insanely Laggy and Spams Client Actions

### ISSUE THREAD

**Issue by RimiNosha** [Bubberstation/Bubberstation#2393]
### Client Version:

515.1637

### Issue Summary:

Tin.

### Round ID:

N/A

### Test Merge Information:

None, raw master.

### Reproduction Steps:

When you open the page, your client will lag like hell, and you'll see a nice little warning in your chat saying you exceeded your actions per second about 120 times. :)

If you click on an item, your client will hang for ~1-3s, and when removing the item, it's somehow worse, and hangs for ~2-6s for me.

It could probably be from my slightly older byond version, but I doubt it.

Bonus screenshot from the server cause I'm too lazy to unadmin myself locally:
![image](https://github.com/user-attachments/assets/f55e9bd6-11ec-4cd2-99f3-36bb47f422ab)

**Comment by JagOfTroy** [comment]
There's also... 119 people connected. That menu runs very very slow with that many people connected.

**Comment by RimiNosha** [comment]
No, this lag is irrespective of connected players. This is local too. Also, as figured out in a TG issue, the action spam is a byond bug, which was fixed in 1638.

**Comment by Shadow-Quill** [comment]
If this was indeed a byond issue that was fixed, then we can close this.
